perf/istio-install/setup_istio_release.sh

```diff
@@ -48,21 +48,21 @@ if [[ "${stream}" == "pre-release" ]];then
   export HELMREPO_URL=https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${release}/charts
   case "${OSTYPE}" in
     darwin*) export RELEASE_URL=https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${release}/istio-${release}-osx.tar.gz ;;
-    linux*) export RELEASE_URL=https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${release}/istio-${release}-linux.tar.gz ;;
+    linux*) export RELEASE_URL=https://gcsweb.istio.io/gcs/istio-prerelease/prerelease/${release}/istio-${release}-linux-amd64.tar.gz ;;
     *) echo "unsupported: ${OSTYPE}" ;;
   esac
 elif [[ "${stream}" == "dev" ]];then
   export HELMREPO_URL=https://gcsweb.istio.io/gcs/istio-build/dev/${release}/charts
   case "${OSTYPE}" in
     darwin*) export RELEASE_URL=https://gcsweb.istio.io/gcs/istio-build/dev/${release}/istio-${release}-osx.tar.gz ;;
-    linux*) export RELEASE_URL=https://gcsweb.istio.io/gcs/istio-build/dev/${release}/istio-${release}-linux.tar.gz ;;
+    linux*) export RELEASE_URL=https://gcsweb.istio.io/gcs/istio-build/dev/${release}/istio-${release}-linux-amd64.tar.gz ;;
```

Comment:
@howardjohn it seems that we are only using this right now. I'm not sure should we change other places or not.

Reply:
Yes, looks like after istio/istio#22381, the `linux.tar.gz` suffix is superseded by `linux-amd64.tar.gz`